packages/pluggableWidgets/datagrid-dropdown-filter-web/src/DatagridDropdownFilter.tsx

```diff
@@ -46,7 +46,7 @@ export default function DatagridDropdownFilter(props: DatagridDropdownFilterCont
                         ariaLabel={props.ariaLabel?.value}
                         attribute={attribute}
                         auto={props.auto}
-                        defaultValue={defaultValues ?? props.defaultValue?.value}
+                        defaultValue={defaultValues.length > 0 ? defaultValues : props.defaultValue?.value}
```

Comment:
Code wise, I understand that we do not want to pass through an empty array here so that's fine.
But I don't quite understand what `const defaultValues = initialFilters.map(filter => filter.value).join(",");` is here in the context of the image that Leo shared in the story. What exactly is value of `defaultValues` here if we apply the scenario from Leo?

Reply:
View state